changelog/update-6157-usability-improvements

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
```

Comment:
@eduardoumpierre we probably need to make sure that this changelog is changed so its not incleded.

Reply:
I think that it won't be included in the final changelog as long as it is a `patch`. Thanks for the comment!